jobs/loc/LocProject.json

```diff
@@ -0,0 +1,14 @@
+{
+    "Projects": [
```

Comment:
In case you haven't already; You'll need to send an email to loc team to get them to add Loc a project for vscode-makefile-tools.  (I can send you their contact info privately).